scripts/load-breaches.js

```diff
@@ -41,17 +41,17 @@ async function handleBreachesResponse(response) {
 }
 
 (async () => {
-  // if (args["--createAMBreach"]) {
-  //   await DBUtils.createBreach({
-  //     Name: "Example Breach Name",
-  //     Domain: "examplebreach",
-  //     BreachDate: "2018-012-06",
-  //     DataClasses: ["Email addresses", "IP addresses", "Passwords", "Usernames", "Website activity"],
-  //     PwnCount: 1234567,
-  //     LogoType: "svg",
-  //     Description: "This is example breach data. Users arrive on this page by clicking the 'Go to Firefox Monitor' button from the Firefox Monitor Add-on (https://github.com/mozilla/blurts-addon) after visiting a site with known data breaches. ",
-  //   });
-  // }
+//   if (args["--createExample"]) {
+//     console.log('here');
+//     await DBUtils.createBreach("Example", {
+//       Name: "Example Breach Name",
+//       Domain: "examplebreachURL",
+//       BreachDate: "2018-012-06",
```

Comment:
nit: i know this is commented out, but we should probably fix the date to "2018-12-06" (not "012")